### PR TITLE
DEVCVL-222 - Bid Component

### PR DESCRIPTION
**Pull request by shakil-adv**
Please watch the attached video for UI demonstration. Thanks

### CODE REVIEW

**Comment by shakil-adv** [comment]
https://user-images.githubusercontent.com/103476595/164147683-12a13d76-ba8a-4b7b-9705-1450d86848d6.mp4

**Comment by jibrail-idris** [comment]
Why are there two sliders?

File: src/components/product/bid/Bid.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                ? renderEndedAuction()
+                : end > now && start < now
+                ? renderStartedAuction()
+                : renderFutureAuction()}
+        </React.Fragment>
+    );
+};
+
+Bid.propTypes = {
+    price: PropTypes.number.isRequired,
```
Comment (kyawthetkt) [inline src/components/product/bid/Bid.js:141]: Could `PropTypes.number.isRequired` accept decimal number such as (0.2)?

**Review by kyawthetkt — APPROVED** [review]

**Comment by shakil-adv** [comment]
> Why are there two sliders?

One for integer values, another for decimal places

**Comment by jibrail-idris** [comment]
> > Why are there two sliders?
> 
> One for integer values, another for decimal places

Couldn't we just do one slider? or just a simple input?

**Comment by shakil-adv** [comment]
> 

Yes we could. But
1. The management wanted a slider
2. A single slider cannot pin point decimal places

To solve that issue, we've two slider.

**Comment by jibrail-idris** [comment]
> 

Yes and let the designer decide if its ok.

**Comment by shakil-adv** [comment]
> Yes and let the designer decide if its ok.
Okay sure, I'll make the changes. Thanks

**Review by jibrail-idris — APPROVED** [review]

**Review by pracheebapate — APPROVED** [review]

**Pull request merged**